### PR TITLE
Javascript-functions completed. Lambert Nguyen

### PR DESCRIPTION
**Pull request by Ln1097**
Screenshot (162)

### CODE REVIEW

File: javascript-functions.tar/javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+console.log('getAreaResult: ', getAreaResult);
+
+function getFirstName(person) {
+  return person.firstName;
+}
+const getFirstNameResult = getFirstName({ firstName: 'Mega', lastName: 'Man' });
+console.log(getFirstNameResult);
+
+function getLastElement(array) {
+  return array[getLastElement.length - 1];
```
Comment (kurtyazdizadeh) [inline javascript-functions.tar/javascript-functions/main.js:26]: this is not correct, `getLastElement` was not a parameter you defined in the function. Instead, you need to get length of the `array` and subtract 1 from it... how would you write that expression?

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Fix the last function `getLastElement`

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Comment by Ln1097** [comment]
Screenshot (163)

**Pull request merged**